### ISSUE TITLE
ActiveControl issue

### ISSUE THREAD

**Issue by JanuszChmiel** [XusinboyBekchanov/VisualFBEditor#6]
I have thought, that when I will use ActiveControl  form property to set focus to some command button, that it will get focus and that will work. But I Am getting The following error.

error 196: PROPERTY has no indexed GET method/accessor 28 
Here is Edited helo.bas with my epxeriment I have tried to add two buttons and I have wanted to make Play button active so it should get focus.
#ifdef __FB_WIN32__
	'#Compile -exx -s gui "Example.rc"
#else
	'#Compile -exx
#endif
#include "mff/Form.bi"
#include "mff/CommandButton.bi"

Using My.Sys.Forms

Dim Shared frm As Form, b1 As CommandButton
Dim b2 As CommandButton

Sub B1_Click(ByRef Sender As Control)

End Sub

B1.Text = "Play"
B1.OnClick = @B1_Click
frm.Add @b1
Sub B2_Click(ByRef Sender As Control)

End Sub
B2.Text = "Stop"
B2.OnClick = @B2_Click
frm.Add @b2
frm.ActiveControl(b1)
frm.Center
frm.Show

App.Run

If there is Windows APi trick or other safe procedure, which would allow Me to make yours GUI toolkit objects focusable, I Am ready to test even bleeding edge code. Really. Yours GUI toolkit is my last hope. Because I have analysed all available Free Basic GUI libraryes and every is too complex and requires adding object position values in Pixels.
I Am not so professional like you are. But I Am sure, that you would find The solution. Now, I can access buttons and other objects only by using NVDA screen reader object navigation functions. But arrow keys, TAB,SHIFT+TAB do not work. It would be really very simple to create useful apps thanks to yours toolkit. But if every button or listbox ETC would had to be accessed by using screen reader object navigation features, if every object would lost focus immediately when I leave main Window by switching to other app and by returning back again, it would not be very blind friendly.
Thank you very much for yours patience.

**Comment by XusinboyBekchanov** [comment]
This is how it works:
```
#ifdef __FB_WIN32__
'#Compile -exx -s gui "Example.rc"
#else
'#Compile -exx
#endif
#include "mff/Form.bi"
#include "mff/CommandButton.bi"

Using My.Sys.Forms

Dim Shared frm As Form, b1 As CommandButton
Dim b2 As CommandButton

Sub B1_Click(ByRef Sender As Control)
	
End Sub

B1.Text = "Play"
B1.OnClick = @B1_Click
frm.Add @b1
Sub B2_Click(ByRef Sender As Control)
	
End Sub
B2.Left = 80
B2.Text = "Stop"
B2.OnClick = @B2_Click
Sub frm_Show(ByRef Sender As Control)
	b1.SetFocus
End Sub
frm.Add @b2
frm.Center
frm.OnShow = @frm_Show
frm.Show

App.Run
```

**Comment by JanuszChmiel** [comment]
Oh yes, thank you very much for yours advice. Yours toolkit is really
outstanding and I have never The chance to find similar one. Very positive
fact is, that all used GUI elements based on textual content are accessible.
So screen reader can get info by using Object navigation by using WIndows
API functions and MSAA calls and GDI32 hooks of NVDA can retrieve all text.
So thank you. I will just try if button get focus and if it will react on
TAB key. Thank you again for yours patience with Me.

**Comment by JanuszChmiel** [comment]
Very well done. Compiled exe shows two focusable buttons. I can press TAB or
SHIFT+TAB and objects are getting focus. Very well done. And The
responsiveness between running app and NVDA is so fast. And NVDA screen
reader even correctly inform Me, if button have been activaed or no. Very
well done. You understand Microsoft WIndows API calls very well. FOrtunately
for Me, yours GUI toolkit uses two fully accessible GUi toolkits. Windows
APi for Windows and GTK for Linux GTK is fully accessible thanks to Orca
screen reader. So thanks to yours GUi, may be, that I would be even able to
create some apps for Linux. I believe, that The code will not differ too
much when I would like to create app for Linux. So very well done, you have
made Me very big joy.
I will be fighting with bass.dll calls. I have finally understood how must I
call functions from this dll. Free Basic compiler contain necessary modules.
I must only pay attention where will I add some code.

You have helped Me very much really. Also thanks to yours .ini file for
Visual editor so I can compile. It is also very good that objects can get
focus so simply. I have been afraid that it would had to be necessary to
call Windows APIS and some complex magic. I only hope, that somebody will
not remove yours framework from Github. I will rather make many backups.
Before I have found yours GUI toolkit, I have been very sad. There was no
simple possibility for Me to create GUI for Free Basic. This was The reason
why I have started with FNX Basic.
But resulting .exe format have been evaluated by Microsoft as potentionally
unreliable. So Windows smart screen have reported warning. So I could not
distribute those exe apps even for Free. No every user would believe Me,
that I Am not distributing malware. people are very sensitive thanks to
various hackers activities. I only hope, tha Microsoft will not classify
machine code created by Free Basic compiler as AN malware. FOrtunately, it
seems to Me, that Free Basic compiler developers are elite programmers,
because creating algorithm which is capable to convert Basic language to
Assembly instructions and create reliable machine code is work for The best
programmers. So thank you very much for yours GUI toolkit.
